Soomla/Assets/Plugins/Soomla/Profile/SoomlaProfile.cs

```diff
@@ -882,14 +893,16 @@ public class SoomlaProfile
 		}
 
 		internal static void TryFireProfileInitialized () {
-			if (AllProvidersInitialized()) {
+			if (AllProvidersInitialized() && nativeModulesInitialized) {
 				ProfileEvents.OnSoomlaProfileInitialized();
 			}
 		}
 
 		/** PROTECTED & PRIVATE FUNCTIONS **/
 
-		protected virtual void _initialize(string customParamsJson) { }
+		protected virtual void _initialize(string customParamsJson) {
+			nativeModulesInitialized = true;
```

Comment:
@borisspektor, we can don't do it, it managed here: https://github.com/vedi/unity3d-profile/blob/muticall-initialize-callbak/Soomla/Assets/Plugins/Soomla/Profile/SoomlaProfile.cs#L45